aio/content/examples/comparing-observables/src/promises.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { docRegionError, docRegionPromise } from './promises';
+
+describe('promises', () => {
+  it('should print 2', (doneFn: DoneFn) => {
+    const consoleLogSpy = spyOn(console, 'log');
+    const pr = docRegionPromise(console, 2);
+    pr.then((value) => {
+      expect(consoleLogSpy).toHaveBeenCalledTimes(1);
+      expect(consoleLogSpy).toHaveBeenCalledWith(2);
+      expect(value).toHaveBeenCalledWith(4);
```

Comment:
You seem to have accidentally changed this while shifting commits around:

```suggestion
      expect(value).toBe(4);
```

Reply:
Sorry about that. I think it should be fixed now.